web/src/main/java/se/inera/intyg/rehabstod/web/controller/api/SickLeaveController.java

```diff
@@ -32,10 +37,17 @@ public class SickLeaveController {
     @Autowired
     private GetActiveSickLeavesService getActiveSickLeavesService;
 
-    @RequestMapping(value = "/active", method = RequestMethod.POST)
-    public SickLeavesResponseDTO getSickLeavesForUnit() {
-        return new SickLeavesResponseDTO(getActiveSickLeavesService.get());
+    @Autowired
+    private PopulateFiltersService populateFiltersService;
+
+    @RequestMapping(value = "/active", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
+    public SickLeavesResponseDTO getSickLeavesForUnit(@RequestBody SickLeavesFilterRequestDTO request) {
+        return new SickLeavesResponseDTO(getActiveSickLeavesService.get(request));
     }
 
+    @RequestMapping(value = "/filters", method = RequestMethod.GET)
```

Comment:
I got a question. I assume that this is the "internal" endpoint that u are using to get filters. 

In IT the endpoint for filters is a @POST (since it's required to send RequestBody). Do we need to change this to a POST or is that logic handled in IntygsTjanstRestIntegrationServiceImpl?

Reply:
This is one should not be affected by the endpoint in IT so we can leave this as a GET.

Reply:
Ok, thanks for clarifying!